tools/pipelines/test-dds-stress.yml

```diff
@@ -36,74 +36,37 @@ parameters:
     - packages/dds/merge-tree
     testFileTarName: sequence
     testCommand: test:stress
+    stageId: sequence_stress_tests
   - name: "@fluid-experimental/tree"
     affectedPaths:
     - experimental/dds/tree
     testFileTarName: experimental-tree
     testCommand: test:stress
+    stageId: experimental_tree_stress_tests
   - name: "@fluidframework/tree"
     affectedPaths:
     - packages/dds/tree
     testFileTarName: tree
     testCommand: test:stress
+    stageId: tree_stress_tests
   - name: "@fluidframework/map"
     affectedPaths:
     - packages/dds/map
     - packages/dds/merge-tree
     testFileTarName: map
     testCommand: test:stress
+    stageId: map_stress_tests
   - name: "@fluidframework/matrix"
     affectedPaths:
     - packages/dds/matrix
     - packages/dds/merge-tree
     testFileTarName: matrix
     testCommand: test:stress
-
+    stageId: matrix_stress_tests
 
 stages:
   - template: templates/include-conditionally-run-stress-tests.yml
     parameters:
       artifactBuildId: $(resources.pipeline.client.runID)
       packages: ${{ parameters.packages }}
       testWorkspace: ${{ variables.testWorkspace }}
-
-  # Capture telemetry about pipeline stages
```

Comment:
All this is now covered by `include-test-real-service.yml`, which `include-conditionally-run-stress-tests.yml` in line 68 already uses.